src/js/main.js

```diff
@@ -317,6 +317,12 @@ if (document.body.dataset.page === "integrations") {
     }
   });
 
+  const params = new URLSearchParams(window.location.search);
+  const integration = params.get("integration");
```

Comment:
Btw I would call this parameter `"type"`, since we are showing a group of integrations, not just a single integration :)